<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 71, 105);
border-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="back_to_my_wordlists_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>20</y>
     <width>281</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;}
QPushButton:hover {background-color: rgb(20, 75, 130);}</string>
   </property>
   <property name="text">
    <string>back to my wordlists</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="words_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>351</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(200, 211, 223);
font: 21pt &quot;Yrsa&quot;;
color:black;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;
selection-color: rgb(101, 145, 232);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yrsa'; font-size:21pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="find_wordlist_input">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>371</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(200, 211, 223);
font: 19pt &quot;Yrsa&quot;;
color:black;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;
selection-color: rgb(101, 145, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="find_wordlist_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>180</y>
     <width>281</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;}
QPushButton:hover {background-color: rgb(20, 75, 130);}</string>
   </property>
   <property name="text">
    <string>find wordlist</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>691</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(225, 225, 225); 
font: 31pt &quot;Yrsa&quot;;</string>
   </property>
   <property name="text">
    <string>Find word lists from other users</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>440</number>
    </property>
    <property name="topMargin">
     <number>270</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>70</number>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
